src/megaclient.cpp

```diff
@@ -13779,27 +13779,94 @@ node_vector MegaClient::getRecentNodes(unsigned maxcount, m_time_t since, bool i
 
 namespace action_bucket_compare
 {
-    std::mutex media_check;   // when we can use c++11, switch to a lambda that remembers the MegaClient* for compare().  In the meantime, force just one MegaClient at a time instead.
-    MegaClient* mc;
-
     const static string webclient_is_image_def = ".jpg.jpeg.gif.bmp.png.";
```

Comment:
Add a comment above this along the lines of: "Don't edit these lists. If you have to then keep them in sync with the webclient!"

Reply:
done!